package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/mozilla/blurts-server/issues"
   },
   "dependencies": {
-    "@sentry/node": "^4.2.3",
+    "@sentry/node": "^5.14.1",
```

Comment:
This one seems to mess up a test in Travis? I added a commit to https://github.com/mozilla/blurts-server/pull/1585 which seems to fix Travis without breaking a test.